packages/components/src/utils/values.js

```diff
@@ -40,3 +40,20 @@ export function isValueEmpty( value ) {
 export function getDefinedValue( values = [], fallbackValue ) {
 	return values.find( isValueDefined ) ?? fallbackValue;
 }
+
+/**
+ * Checks to see if a value is a numeric value (`number` or `string`).
+ *
+ * @param {any} value
+ *
+ * @return {boolean} Whether value is numeric.
+ */
+export function isValueNumeric( value ) {
+	const obj = typeof value === 'string' ? value.replace( /,/g, '' ) : value;
+	return (
+		! isNaN( parseFloat( obj ) ) &&
+		! isNaN( Number( obj ) ) &&
```

Comment:
Do we actually need to pass it to `Number` first? Doesn't `isNaN` already coerce the value into a `Number`?

Reply:
> isNaN already coerce the value into a Number

I believe it does. [Source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN)

This was a copy paste from G2. Happy to refactor here or in another PR.